app/config.ts

```diff
@@ -2,6 +2,7 @@ const config = {
   clientId: '{ClientID}',
   domain: '{DomainName}',
   auth0Scope: 'openid profile email',
+  audience: '{yourApiIdentifier}',
```

Comment:
What is this new config for?
We should probably add docstrings for each of this field.

Reply:
What kind of docstring do you have in mind? Reference link to auth0?

Reply:
Something like the following. I added an example for clientId. Just fill in the docstrings for the rest.
```ts
/**
 * Auth0 related configurations
 */
const config = {
  /**
   * The unique identifier for your Auth0 application.
   * See https://auth0.com/docs/get-started/applications/application-settings#basic-settings
   */
  clientId: '{ClientID}',
  domain: '{DomainName}',
  auth0Scope: 'openid profile email',
  audience: '{yourApiIdentifier}',
}
```

Reply:
I understand we don't do this in most part of the code. But I think we should start doing it on areas where things are not self explanatory. For example on this new config, I'm not sure what audience really means :D